docs/howto/system_testing.md

```diff
@@ -39,19 +38,33 @@ Packages have a specific folder structure (only relevant parts shown).
   manifest.yml
 ```
 
-To define a system test we must define configuration at two levels: the package level and each data stream's level.
+To define a system test we must define configuration on at least one level: a package or a data stream's one.
 
-### Package-level configuration
-
-First, we must define the configuration for deploying a package's integration service. As mentioned in the [_Limitations_](#Limitations) section above, only packages whose integration services can be deployed via Docker Compose are supported at the moment.
+First, we must define the configuration for deploying a package's integration service. We can define it on either the package-level:
 
 ```
 <package root>/
   _dev/
     deploy/
-      docker/
-        docker-compose.yml
+      <service deployer>/
+        <service deployer files>
+```
+
+or the data-stream's level:
+
 ```
+<package root>/
+  data_stream/
+    <data stream>/
+      _dev/
+        deploy/
+          <service deployer>/
+            <service deployer files>
+```
+
+`<service deployer>` - a name of the supported service deployer: `docker` or `tf` (terraform).
+
+### Docker Compose service deployer
 
 The `docker-compose.yml` file defines the integration service(s) for the package. If your package has a logs data stream, the log files from your package's integration service must be written to a volume. For example, the `apache` package has the following definition in it's integration service's `docker-compose.yml` file.
```

Comment:
This sentence made sense earlier because the sample file structure above showed a `docker-compose.yml` file. Now the sample file structure above just says `<service deployer files>` so this sentence doesn't make as much sense.

I would add another introductory sentence to bridge this gap, something like:

> When using the Docker Compose service deployer, the `<service deployer files>` must include a `docker-compose.yml` file.

Reply:
Added here and for the TF service deployer